std/typecons.d

```diff
@@ -4009,6 +4015,24 @@ nothrow pure @nogc @safe unittest
     assert(result.get == 4);
 }
 
+// test that Nullable.get(default) can merge types
+@safe @nogc nothrow pure
+unittest
+{
+    Nullable!ubyte sample = Nullable!ubyte();
+
+    assert(sample.get(1000) == 1000);
+}
+
+// regression test for workaround for https://issues.dlang.org/show_bug.cgi?id=20670
```

Comment:
```suggestion
// Workaround for https://issues.dlang.org/show_bug.cgi?id=20670
```

Technically all unittests are regression tests (but not all of them are tests for regressions).

Reply:
done